plugin-enclave-gradle/src/main/kotlin/com/r3/conclave/plugin/enclave/gradle/gramine/GenerateGramineBundle.kt

```diff
@@ -85,6 +89,61 @@ open class GenerateGramineBundle @Inject constructor(
         }
     }
 
+    private fun findJavaModules(): String {
+        val enclaveJar = outputDir.file(GRAMINE_ENCLAVE_JAR).get().asFile.absolutePath
+
+        val dependentModules = execCommand(
+            "jdeps",
+            "--print-module-deps",
+            "--ignore-missing-deps",
+            enclaveJar
+        ).run { trimEnd().replace(System.lineSeparator(), "").split(",") }
+        // TODO: Provide mechanism to allow the user to add modules needed when using reflection
+        val defaultModules = listOf("java.base", "java.desktop", "java.sql", "jdk.crypto.ec")
```

Comment:
Can you elaborate more on why these modules are needed for reflection.

Reply:
`java.desktop` and `java.sql` are needed by integration tests. `jdk.crypto.ec` are needed in the context of attestation (only debug and release mode affected). 
If I recollect correctly this is where it was failing
``
`fun ByteCursor<SgxEcdsa256BitPubkey>.toPublicKey(): PublicKey {
    val encodedKey = ByteBuffer.allocate(P256_HEAD.size + this.size)
    encodedKey.put(P256_HEAD)
    encodedKey.put(this.buffer)
    val keySpec = X509EncodedKeySpec(encodedKey.array())
    return KeyFactory.getInstance("EC").generatePublic(keySpec)
}
```

Reply:
As discussed offline, let's add `jdk.crypto.ec` in the SDK, and add a config to allow the user to add extra modules for their enclaves.

Reply:
This is done